.github/workflows/integration-test.yml

```diff
@@ -16,30 +19,25 @@ jobs:
       with:
         python-version: 3.6.10
 
-    # # to cache and restore the download data and python packages.
-    # # still have bug (Cache not found for input keys)
-    # # leave the issue to another PR
-    # - uses: actions/cache@v2
-    #   id: cache
-    #   with:
-    #     path: |
-    #       /home/runner/work/nn-Meter/data/testmodels
-    #       /home/runner/.nn_meter/data
-    #       # /opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6
-    #     key: ${{hashFiles('setup.py')}}-${{hashFiles('tests/requirements.txt') }}
-    #     # location remider:
-    #     # pwd: /home/runner/work/nn-Meter/nn-Meter
-    #     # python /opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages
-    #     # pythonLocation: /opt/hostedtoolcache/Python/3.6.10/x64
-    #     # LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.6.10/x64/lib
-    #     # package path: /opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages
+    - name: Cache
+      uses: actions/cache@v2
+      id: cache
+      env: 
+        cache-name: download-cache
+      with:
+        path: |
+          ~/.nn_meter
+          /home/runner/work/nn-Meter/data/testmodels
+          /opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages
```

Comment:
make sure your installation of `nn-meter` is the latest by 
1. uninstall `pip uninstall nn-meter` before installation 
or 
2. use `pip install -U .` to make sure replace existing if any
or 
3. (not recommended) use `pip install --user` to install to another location

Reply:
In method 1, the argument `-y` should be applied to avoid the confirmation of uninstall deletions. In the current version, method 2 is used.